api/docs/release.dox

```diff
@@ -232,6 +232,8 @@ Further non-compatibility-affecting changes include:
    insertion of instrumentation and instrumentation optimizations. Its primary purpose
```

Comment:
PR description:
> Implements drstatecmp, a library for machine state comparison for detecting instrumentation-induced clobbering of

nit: This is a really wide line width to have inserted newlines: I think people would expect either 80-char or maybe 90-char, or else no newlines at all.

Reply:
You are right it is too long. Changed it